data/meson.build

```
@@ -0,0 +1,15 @@
+systemd_units = [
+  'hexagonrpcd-adsp-rootpd.service',
+  'hexagonrpcd-adsp-sensorspd.service',
+  'hexagonrpcd-sdsp.service',
+]
+
+foreach unit : systemd_units
+  configure_file(
+    input: unit + '.in',
+    output: unit,
+    configuration: { 'bindir': get_option('prefix') / get_option('bindir') },
+    install: true,
+    install_dir: get_option('prefix') / 'lib' / 'systemd' / 'system',
```

```suggestion
    install_dir: get_option('libdir') / 'systemd/system',
```

The `/` operator is for concatenating paths, but no need for Meson to concatenate literal paths. Also, bindir and libdir should be absolute paths already (unless there's a special case).

ModemManager seems to use `dependency('systemd').get_pkgconfig_variable('systemdsystemunitdir')`, but that could take more review time.